addons/im_livechat/static/src/core/thread_model_patch.js

```diff
@@ -69,7 +69,7 @@ patch(Thread.prototype, "im_livechat", {
         if (this.type !== "livechat") {
             return this._super();
         }
-        return this.correspondent && !this.correspondent.is_public
+        return this.correspondent && this.correspondent.type !== "guest"
```

Comment:
remove livechat override?

Reply:
The part about the guest is removed, the part about `user_livechat_username` is kept :+1:

Reply:
Guest is not yet handled in `thread.imgUrl` since we cannot open chats with guests. Let's do it later (+ it's a change in the behavior of the livechat)